### PR TITLE
Pin sra-toolkit version in resolwebio/utils Dockerfile

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -51,6 +51,9 @@ Fixed
   BBDuk - STAR - featureCounts workflow
 - Make ``cuffnorm`` process attach correct expression data objects to
   samples
+- Pin ``sra-toolkit`` version to 2.9.0 in ``resolwebio/utils`` Docker image.
+  Before, "current" version was downloaded which caused that automated Docker
```
Comment (tjanez) [inline docs/CHANGELOG.rst]: Please, move the explanation of why this change was made to commit message's body and just note the change in the CHANGELOG without explanation.

File: resolwe_bio/docker_images/utils/packages-manual/sra-toolkit.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ download_and_verify \
     sra-toolkit \
     2.9.0 \
     911301a6f85bdbf8e786a6b6561d32f262cd356ae52acd27830e39fb07e41b12 \
-    https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-ubuntu64.tar.gz \
+    https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/2.9.0/sratoolkit.\${version}-ubuntu64.tar.gz \
```
Comment (mzganec) [inline resolwe_bio/docker_images/utils/packages-manual/sra-toolkit.sh]: One more time.
